src/test/java/gov/loc/repository/bagit/verify/BagVerifierTest.java

```diff
@@ -118,6 +120,10 @@ public void testVersion0_97IsValid() throws Exception{
   @Test
   public void testVersion0_98IsValid() throws Exception{
     rootDir = Paths.get(getClass().getClassLoader().getResource("bags/v0_98/bag").toURI());
+    if (System.getProperty("os.name").contains("Windows")){
+    	Path bagitDir = rootDir.resolve(".bagit");
```

Comment:
the .bagit folder does not have to be hidden. .bagit is our incubating next version of the bagit spec; it solves a lot of problems from the originial spec.